src/saving/SaveList.cs

```diff
@@ -195,6 +217,29 @@ private void OnInvalidLoaded(string saveName)
         loadInvalidConfirmDialog.PopupCenteredMinsize();
     }
 
+    private void OnInvalidOpenLog(string saveName)
+    {
+        var path = ProjectSettings.GlobalizePath($"user://logs/{saveName}.error.log.txt");
+        Process.Start(path);
+    }
+
+    private void OnInvalidCancel()
+    {
+        loadInvalidConfirmDialog.Hide();
+    }
+
+    private void OnConfirmLoadOlder()
+    {
+        Save.LogErrorToFile(saveToBeLoaded, "OLD", null);
```

Comment:
This would just be then GD.PrintErr("loading older save or some stuff like that")